code/modules/clothing/spacesuits/rig/modules/specific/ai_container.dm

```diff
@@ -213,3 +228,237 @@
 		to_chat(user, "<span class='warning'>There is no active AI within \the [ai].</span>")
 	update_verb_holder()
 	return
+
+/*
+ * The realm of cyberpunk, a Neural Jack-enabled IIS.
+ * Components unashamedly utilizing the groundwork of borers.
```

Comment:
Oh god. This got an actual out loud despairing 'Noooo.' out of me.
Borers are a fractal nightmare of barely functional code and problematic hacks. Ominous.